core/src/avm2/globals/globals.as

```diff
@@ -24,6 +24,19 @@ include "flash/utils/ByteArray.as"
 include "flash/utils/Dictionary.as"
 include "flash/desktop/ClipboardFormats.as"
 include "flash/desktop/ClipboardTransferMode.as"
+
+include "flash/events/IEventDispatcher.as"
```

Comment:
Is this not alphabetically-sorted for a reason?
If it does, please explain it in a comment (what other classes depend on it).
If not, please move it next to the other `flash.events` classes.

Reply:
> Is this not alphabetically-sorted for a reason?

Interface before the class using it.
We never had them fully ordered and it's common enough to have dependency ordering that I'm not sure if it requires a comment.

Reply:
I'd say it's less so that it doesn't need a comment because it's common, but more so that it doesn't need a comment because the comment at the top of the file explains this situation, so another one in each location where this occurs seems unnecessarily verbose:
https://github.com/ruffle-rs/ruffle/blob/559ec8356ddf909d2bc0ca6561d2fff334eb3151/core/src/avm2/globals/globals.as#L1-L2.

Reply:
That comment mentions superclasses, not interface/class dependencies.

Reply:
I've expanded the comment to mention interfaces